src/store/transactions/force-exit/force-exit.actions.ts

```diff
@@ -0,0 +1,190 @@
+import { BigNumber } from "@ethersproject/bignumber";
+
+// domain
+import { Account, PoolTransaction } from "src/domain/hermez";
+import { Accounts } from "src/persistence";
+
+export enum ForceExitActionTypes {
+  GO_TO_CHOOSE_ACCOUNT_STEP = "[FORCE EXIT] GO TO CHOOSE ACCOUNT STEP",
+  GO_TO_BUILD_TRANSACTION_STEP = "[FORCE EXIT] GO TO BUILD TRANSACTION STEP",
+  GO_TO_REVIEW_TRANSACTION_STEP = "[FORCE EXIT] GO TO REVIEW TRANSACTION STEP",
+  LOAD_ACCOUNTS = "[FORCE EXIT] LOAD ACCOUNTS",
+  LOAD_ACCOUNTS_SUCCESS = "[FORCE EXIT] LOAD ACCOUNTS SUCCESS",
+  LOAD_ACCOUNTS_FAILURE = "[FORCE EXIT] LOAD ACCOUNTS FAILURE",
+  LOAD_POOL_TRANSACTIONS = "[FORCE EXIT] LOAD POOL TRANSACTIONS",
+  LOAD_POOL_TRANSACTIONS_SUCCESS = "[FORCE EXIT] LOAD POOL TRANSACTIONS SUCCESS",
+  LOAD_POOL_TRANSACTIONS_FAILURE = "[FORCE EXIT] LOAD POOL TRANSACTIONS FAILURE",
+  START_TRANSACTION_APPROVAL = "[FORCE EXIT] START TRANSACTION APPROVAL",
+  STOP_TRANSACTION_APPROVAL = "[FORCE EXIT] STOP TRANSACTION APPROVAL",
+  RESET_STATE = "[FORCE EXIT] RESET STATE",
+}
+
+export type Step = "choose-account" | "build-transaction" | "review-transaction";
+
+export interface TransactionToReview {
+  amount: BigNumber;
+  fee: number;
+  from: Account;
+  to: Partial<Account>;
+}
+
+export interface GoToChooseAccountStep {
+  type: ForceExitActionTypes.GO_TO_CHOOSE_ACCOUNT_STEP;
+}
+
+export interface GoToBuildTransactionStep {
+  type: ForceExitActionTypes.GO_TO_BUILD_TRANSACTION_STEP;
+  account?: Account;
```

Comment:
Right, changed to `null` to avoid having the `undefined`. In fact, we need to change some properties from some reducers that could be `undefined` to `null` also. I'll try to do it in another PR.

Reply:
Wait, why null? The `account` should also not be null in the payload of the action. It should be always an `Account`.